src/Atmos/Model/AtmosModel.jl

```diff
@@ -716,4 +718,107 @@ function init_state_conservative!(
 )
     m.init_state_conservative(m, state, aux, coords, t, args...)
 end
+
+struct RoeNumericalFlux <: NumericalFluxFirstOrder end
+function numerical_flux_first_order!(
+    numerical_flux::RoeNumericalFlux,
+    balance_law::AtmosModel,
+    fluxᵀn::Vars{S},
+    normal_vector::SVector,
+    state_conservative⁻::Vars{S},
+    state_auxiliary⁻::Vars{A},
+    state_conservative⁺::Vars{S},
+    state_auxiliary⁺::Vars{A},
+    t,
+    direction,
+) where {S, A}
+
+    numerical_flux_first_order!(
+        CentralNumericalFluxFirstOrder(),
+        balance_law,
+        fluxᵀn,
+        normal_vector,
+        state_conservative⁻,
+        state_auxiliary⁻,
+        state_conservative⁺,
+        state_auxiliary⁺,
+        t,
+        direction,
+    )
+
+    FT = eltype(fluxᵀn)
+    param_set = balance_law.param_set
+    _cv_d::FT = cv_d(param_set)
```

Comment:
You don't need ` _cv_d`, `_T_0` and `γ` here if you use the Thermodynamics functions below (which properly generalize to the moist case).